Multisig/Logic/Models/Transaction.swift

```diff
@@ -90,7 +111,24 @@ extension Transaction {
         return transaction
     }
 
-    func safeTxHash(by safeAddress: Address) -> HashString? {
+    private var safeEncodedTxData: Data {
```

Comment:
might be different for different versions of the safe contract

Reply:
not before 1.3.0